dao/searchDao.js

```diff
@@ -0,0 +1,14 @@
+const db = require('./db');
+
+exports.getAll = function(filter, cb) {
+    let classFilter = '';
+    if(filter.class) {
+        classFilter = ' AND class = ' + db.escape(filter.class);
```

Comment:
The way I had imagined these working was that it would be `class IN (comma-delimited-values)`, or perhaps in this case `class > base` or `class < base`; what if someone is willing to fly coach or business-class but not first class (or first-class or business-class but not economy-class)?

Reply:
Yes, you are correct.

Reply:
But how would that work for the parameter being passed in?

Reply:
Depending on whether the user wanted "nothing less than" or "nothing more expensive than", `AND class >= ' + db.escape(filter.class)` or `AND class <= ' + db.escape(filter.class)` would do. But if you don't want to complicate the code with this feature, you can leave it be.